integ-test/src/test/java/org/opensearch/sql/ppl/SimpleQueryStringIT.java

```diff
@@ -20,36 +19,42 @@ public void init() throws IOException {
 
   @Test
   public void test_simple_query_string() throws IOException {
-    String query = "SOURCE=" + TEST_INDEX_BEER
-        + " | WHERE simple_query_string([\\\"Tags\\\" ^ 1.5, Title, 'Body' 4.2], 'taste') | fields Id";
+    String query =
+        "SOURCE="
+            + TEST_INDEX_BEER
+            + " | WHERE simple_query_string([\\\"Tags\\\" ^ 1.5, Title, 'Body' 4.2], 'taste') |"
+            + " fields Id";
     var result = executeQuery(query);
     assertEquals(16, result.getInt("total"));
   }
 
   @Test
   public void test_simple_query_string_all_params() throws IOException {
-    String query = "SOURCE=" + TEST_INDEX_BEER
-        + " | WHERE simple_query_string(['Body', Tags, Title], 'taste beer', default_operator='or',"
-        + "analyzer=english, analyze_wildcard = false, quote_field_suffix = '.exact',"
-        + "auto_generate_synonyms_phrase_query=true, boost = 0.77, flags='PREFIX',"
-        + "fuzzy_transpositions = false, lenient = true, fuzzy_max_expansions = 25,"
-        + "minimum_should_match = '2<-25% 9<-3', fuzzy_prefix_length = 7) | fields Id";
+    String query =
+        "SOURCE="
+            + TEST_INDEX_BEER
+            + " | WHERE simple_query_string(['Body', Tags, Title], 'taste beer',"
+            + " default_operator='or',analyzer=english, analyze_wildcard = false,"
+            + " quote_field_suffix = '.exact',auto_generate_synonyms_phrase_query=true, boost ="
```

Comment:
Can you reformat to have line breaks between args only (on comma)?

Reply:
Don't believe we can, as next `spotlessApply` will reformat the string.

Reply:
Resolved.